extensions/amp-story/0.1/system-layer.js

```diff
@@ -174,7 +174,7 @@ export class SystemLayer {
     this.root_ = renderAsElement(this.win_.document, TEMPLATE);
 
     this.root_.insertBefore(
-        this.progressBar_.build(pageCount), this.root_.firstChild);
+        this.progressBar_.build(pageCount), this.root_.firstChild.nextSibling);
```

Comment:
What element is this? Can you make this clear by moving to a local var?

Reply:
Does this change anything on mobile as a side-effect?

Reply:
@alanorozco nups, mobile is all absolutely positioned

Reply:
@choumx Done